packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/ColumnComponent.java

```diff
@@ -85,6 +86,18 @@ private Element doRender(VariableManager variableManager, Object object) {
                                 .findFirst()
                                 .ifPresent(columnElementProps::width));
 
+        this.props.tableEvents().stream()
```

Comment:
You can compute that before the creation of the columnElementProps. This way, you would not split the builder in multiple parts which gives the opportunity to forget some properties.